runtime/parachains/src/configuration.rs

```diff
@@ -1079,11 +1095,11 @@ mod tests {
 				hrmp_recipient_deposit: 4905,
 				hrmp_channel_max_capacity: 3921,
 				hrmp_channel_max_total_size: 7687,
-				hrmp_max_parachain_inbound_channels: 3722,
-				hrmp_max_parathread_inbound_channels: 1967,
+				hrmp_max_parachain_inbound_channels: 37,
+				hrmp_max_parathread_inbound_channels: 19,
```

Comment:
Is there any specific justification for the values chosen, or did we just want to have a random value less than the bound?